pkg/project/auth/cache_test.go

```diff
@@ -0,0 +1,143 @@
+package auth
+
+import (
```

Comment:
> I need the Fake client for PolicyBindings and Policy objects to actually support returning a Fake value

I wasn't planning it, but I'm not necessarily opposed.  It doesn't look like any of the Fake clients in origin or in kubernetes do that.  Do you have some some example of what you want?